Detections/OfficeActivity/Office_MailForwarding.yaml

```diff
@@ -19,49 +19,28 @@ relevantTechniques:
   - T1114
   - T1020
 query: |
-
+  let queryfrequency = 1d;
+  let queryperiod = 7d;
   OfficeActivity
-  | where Operation =~ "Set-Mailbox"
-  | where Parameters has "ForwardingSmtpAddress"
-  | extend parsed = parse_json(Parameters)
-  | mv-expand parsed
-  | where parsed.Name == "ForwardingSmtpAddress"
-  | extend parameterName = tostring(parsed.Name), fwdingDestination = tostring(parsed.Value)
-  | where isnotempty(fwdingDestination)
-  | extend ClientIPOnly = case( 
-  ClientIP has "." and ClientIP has ':', tostring(split(ClientIP,":")[0]), 
-  ClientIP has "." and ClientIP has '-', tostring(split(ClientIP,"-")[0]), 
-  ClientIP has ']-', tostring(trim_start(@'[[]',tostring(split(ClientIP,"]")[0]))),
-  ClientIP has ']:', tostring(trim_start(@'[[]',tostring(split(ClientIP,"]")[0]))),
-  isempty(ClientIP) and ClientIP_ has "." and ClientIP_ has ':', tostring(split(ClientIP_,":")[0]), 
-  isempty(ClientIP) and ClientIP_ has "." and ClientIP_ has '-', tostring(split(ClientIP_,"-")[0]), 
-  isempty(ClientIP) and ClientIP_ has ']-', tostring(trim_start(@'[[]',tostring(split(ClientIP_,"]")[0]))),
-  isempty(ClientIP) and ClientIP_ has ']:', tostring(trim_start(@'[[]',tostring(split(ClientIP_,"]")[0]))),
-  isnotempty(ClientIP), ClientIP,
-  isnotempty(ClientIP_), ClientIP_,
-  "IP Not Available"
-  )  
-  | extend Port = case(
-  ClientIP has "." and ClientIP has ':', tostring(split(ClientIP,":")[1]), 
-  ClientIP has "." and ClientIP has '-', tostring(split(ClientIP,"-")[1]), 
-  ClientIP has ']-', tostring(split(ClientIP,"]-")[1]), 
-  ClientIP has ']:', tostring(split(ClientIP,"]:")[1]), 
-  isempty(ClientIP) and ClientIP_ has "." and ClientIP_ has ':', tostring(split(ClientIP_,":")[1]), 
-  isempty(ClientIP) and ClientIP_ has "." and ClientIP_ has '-', tostring(split(ClientIP_,"-")[1]), 
-  isempty(ClientIP) and ClientIP_ has ']-', tostring(split(ClientIP_,"]-")[1]),
-  isempty(ClientIP) and ClientIP_ has ']:', tostring(split(ClientIP_,"]:")[1]),
-  isnotempty(ClientIP), ClientIP,
-  isnotempty(ClientIP_), ClientIP_,
-  "IP Not Available"
-  )
-  | extend UserId = iff(isempty(UserId), UserId_, UserId)
-  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), DistinctUserCount = dcount(UserId), UserId = make_set(UserId), 
-  Ports = make_set(Port), EventCount = count() by fwdingDestination, ClientIP = ClientIPOnly 
-  | where DistinctUserCount > 1
+  | where TimeGenerated > ago(queryperiod)
+  //| where OfficeWorkload =~ "Exchange"
```

Comment:
Is there a reason to comment out this line?

Reply:
@petebryan Maintenance of these two versions of the rule should be as easy as possible, so I preferred to keep both versions as identical as possible. This condition ```| where Operation in ("Set-Mailbox", "New-InboxRule", "Set-InboxRule")``` is not possible in the NRT version, so the condition ```"| where OfficeWorkload =~ "Exchange"``` had to be  added in the first NRT version and did not exist in the scheduled version. As this line was not strictly needed in the scheduled version, I kept it commented.

After adding the condition ```| where Parameters has_any ("ForwardTo", "RedirectTo", "ForwardingSmtpAddress")``` I think the commented line should instead be ```| where Operation in ("Set-Mailbox", "New-InboxRule", "Set-InboxRule")```. I think both versions should try to be similar, if you think this line should be uncommented or deleted, let me know, feel free to commit any change.